lab11_1/src/main.js

```diff
@@ -2,16 +2,23 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import { useState } from 'react';
 
-const App = () => {
+import './style.css';
 
+const MultiInput = () => {
+  const [inputVal, setInputVal] = useState('type something...');
   return (
-    <div>
-      <p>The largest counter is: ... </p>
+    <div id="app-container">
+      {[...Array(Number(5)).keys()].map(i => (

```

Comment:
There is no need for destructuring or `Number` or `keys`, your idea is correct, and you can implement it in a much simpler way: `Array(5).map(...`

Reply:
You mean:

```
Array(5).fill(null).map((x, i) => i)
```

Reply:
@ynonp thanks , learned something new
but i would like to know why is your suggested method is better then this
`[...Array(5).keys()]`
?

Reply:
After checking again performance is actually much better using [...Array(5).keys()]. I was referring to @ilyador suggestion because I thought he was missing the call to .fill()
(I checked here https://jsbench.me/)

In any case and in any real world situation I'd probably use _.range(5) from the lodash or underscore library

Reply:
@ynonp Why use fill? using `Array(x)` creates X `undefined` items, so there is no actual difference between this and `.fill(null)` I think.

>After checking again performance is actually much better using [...Array(5).keys()]

Really? 😆 adding an extra action of restructuring makes it faster??